department-of-veterans-affairs/caseflow
# 8439 catch promise error

Resolves #8439 
cc: @lomaxap 
### Description
catch unhandled promise errors in the `requestSave` method

### Acceptance Criteria

### Testing Plan
1. open up two tabs and make your way to this url: http://localhost:3000/queue/appeals/3626186/tasks/3626186/draft_decision/dispositions/edit/2
2. Try to edit the same disposition that you've already deleted in another tab
3. you should see an error message appear
4. it should not be sent to sentry any longer since it is caught. (is there any way to verify that it's not sent to Sentry, before this PR is merged?).
